src/app/providers/slot-selector/slot-selector.ts

```diff
@@ -30,7 +32,7 @@ export class SlotSelectorProvider {
     return slotItems.map((slotItem) => {
       return {
         ...slotItem,
-        component: this.resolveComponentName(), // slotItem
+        component: this.resolveComponentName(slotItem), // slotItem
```

Comment:
not sure if this small comment is needed

Reply:
Yep fair enough, will remove